RELEASE/scripts/autoscend/auto_quest_level_12.ash

```diff
@@ -192,10 +140,10 @@ int auto_warAdvReqCB()
 	}
 	
 	float expectedItemDropMulti = 1 + simValue("Item Drop")/100;
-	return ceil(20.0 / expectedItemDropMulti);
+	return 4*ceil(20.0 / expectedItemDropMulti);
```

Comment:
```suggestion
	return 4*ceil(5.0 / expectedItemDropMulti);
```

This is a count of how many adventures it will take. 20 is the starting position because 1/4 chance to encounter and 1/5 chance to drop multiply together to 20 adventures to get. then reduced by by drop rate.

However, I forgot to account for the possibility that you get 400%+ item drop bonus in which case it will take only 4 adv. moving the 4* outside the ceil function would do it. But to properly move it out requires replacing the 20.0 with a 5.0
As it is, you are saying that it would take 80 adventures to farm with +0% item drop which is not true.

Reply:
Yeah, added the item drop cap and made the code more readable in b16fae0.

Reply:
new code looks good